README.md

```diff
@@ -287,70 +299,70 @@ grunt.initConfig({
 });
 ```
 
-#### Sample RequireJS/NPM Template usage
+#### Passing options to sandbox (puppeteer)
+
+See [puppeteer launch options](https://pptr.dev/#?product=Puppeteer&version=v3.0.1&show=api-puppeteerlaunchoptions) for a complete list of arguments.
 
 ```js
 // Example configuration
 grunt.initConfig({
   jasmine: {
-    yourTask: {
+    customTemplate: {
       src: 'src/**/*.js',
       options: {
         specs: 'spec/*Spec.js',
-        template: require('grunt-template-jasmine-requirejs')
+        helpers: 'spec/*Helper.js',
+        template: 'custom.tmpl',
+        sandboxArgs: {
+          args: ['--no-sandbox'],
+          timeout: 3000,
+          defaultViewport: {
+            isMobile: true
+          }
+        }
       }
     }
   }
 });
 ```
 
-NPM Templates are just node modules, so you can write and treat them as such.
-
-Please see the [grunt-template-jasmine-requirejs](https://github.com/jsoverson/grunt-template-jasmine-requirejs) documentation
-for more information on the RequireJS template.
-
-#### Keeping temp files in an existing directory
-
-Supplying a custom temp directory
+#### Sample RequireJS/NPM Template usage
 
 ```js
 // Example configuration
 grunt.initConfig({
   jasmine: {
-    pivotal: {
+    yourTask: {
       src: 'src/**/*.js',
       options: {
-        keepRunner: true,
-        tempDir: 'bin/jasmine/',
         specs: 'spec/*Spec.js',
-        helpers: 'spec/*Helper.js'
+        template: require('grunt-template-jasmine-requirejs')
       }
     }
   }
 });
 ```
 
-#### Passing options to sandbox (puppeteer)
+NPM Templates are just node modules, so you can write and treat them as such.
 
-See [puppeteer launch options](https://pptr.dev/#?product=Puppeteer&version=v3.0.1&show=api-puppeteerlaunchoptions) for a complete list of arguments.
+Please see the [grunt-template-jasmine-requirejs](https://github.com/jsoverson/grunt-template-jasmine-requirejs) documentation
+for more information on the RequireJS template.
+
+#### Keeping temp files in an existing directory
+
+Supplying a custom temp directory
 
 ```js
 // Example configuration
 grunt.initConfig({
   jasmine: {
-    customTemplate: {
+    pivotal: {
       src: 'src/**/*.js',
       options: {
+        keepRunner: true,
+        tempDir: 'bin/jasmine/',
         specs: 'spec/*Spec.js',
-        helpers: 'spec/*Helper.js',
-        template: 'custom.tmpl',
-        sandboxArgs: {
-          args: ['--no-sandbox'],
-          timeout: 3000,
-          defaultViewport: {
-            isMobile: true
-          }
-        }
```

Comment:
i don't think you should remove the sandbox args from the sandbox arg demo ;)

Reply:
oh i see you moved them to an examples file?

Reply:
yeap!